server/planning/common.py

```diff
@@ -223,18 +223,52 @@ def remove_lock_information(item):
     item.update({LOCK_USER: None, LOCK_SESSION: None, LOCK_TIME: None, LOCK_ACTION: None})
 
 
-def get_coverage_cancellation_state():
+def get_default_coverage_status_qcode_on_ingest(current_app=None):
+    return (current_app or app).config.get("PLANNING_DEFAULT_COVERAGE_STATUS_ON_INGEST", "ncostat:int")
+
+
+def get_coverage_status_from_cv(qcode: str):
+    default_states = {
```

Comment:
is the coverage state mandatory? if not I would rather keep it empty than using some hardcoded values, or we might need to add those to settings

Reply:
In Planning front-end we're always setting a value, and we have business logic in the back-end involving coverage state. So I would say yes it is mandatory.

I updated the code so it raises an error instead